client/lib/domains/types.ts

```diff
@@ -104,7 +106,7 @@ export type ResponseDomain = {
 	registryExpiryDate: string;
 	sslStatus: string | null;
 	subscriptionId: string | null;
-	subdomainPart: string;
+	subdomainPart?: string;
```

Comment:
Super NIT: Should not go `subdomainPart` before `subscriptionId`? (alphabetically speaking)

Reply:
Good catch! I just fixed this.